src/main/java/org/tinder/filters/MessagesStaticResourcesFilter.java

```diff
@@ -0,0 +1,37 @@
+package org.tinder.filters;
+
+import org.tinder.services.Services;
+
+import javax.servlet.RequestDispatcher;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+
+public class MessagesStaticResourcesFilter extends RequestFilter{
```

Comment:
can u create this filter for all path in app (/*)

Reply:
ok